src/components/CountdownComponent/CountdownComponent.tsx

```diff
@@ -1,5 +1,60 @@
-function CountdownComponent() {
-  return <h1>Countdown component</h1>;
-}
+import React, { useState, useEffect } from 'react';
+
+// type of calculateTimeLeft
+type TimeLeft = {
+  days: number;
+  hours: number;
+  minutes: number;
+  seconds: number;
+};
+
+// a functional component
+const CountdownComponent: React.FC = () => {
+  // const targetDate = '2025-04-09T23:59:59'; // April 10th is MIDI
+  const targetDate = import.meta.env.VITE_MIDI_DATE;
+  console.log("targetDate: " + targetDate);
+
+  // helper function
+  const calculateTimeLeft = () : TimeLeft => {
```

Comment:
Tarpo po skliausteliu ties dvitaskiu nereikia
```suggestion
  const calculateTimeLeft = (): TimeLeft => {
```